share/spack/gitlab/cloud_pipelines/stacks/e4s-cray-sles/spack.yaml

```diff
@@ -0,0 +1,80 @@
+spack:
+  view: false
+
+  concretizer:
+    reuse: false
+    unify: false
+
+  compilers:
```

Comment:
Since these are tied to the tags for this runner, which are noted in the `cray-sles/zen4/ci.yaml` it may make sense to move the compiler configs next to the ci configs for these runners.